package.json

```diff
@@ -34,9 +34,9 @@
     "coverage": "grunt coverage --mocha-reporter=dot"
   },
   "dependencies": {
+    "backbone": "^1.2.1",
     "jquery": "^1.8.0 || ^2.1.0",
-    "backbone": "1.0.0 - 1.1.2",
-    "underscore": "1.4.4 - 1.6.0"
+    "underscore": "^1.8.3"
```

Comment:
You've gotta believe it'll get better

On Wed, Jul 15, 2015 at 12:19 PM, Sam Saccone <notifications@github.com>
wrote:

> In package.json
> <https://github.com/marionettejs/backbone.syphon/pull/114#discussion_r34697063>
> :
>
> >      "jquery": "^1.8.0 || ^2.1.0",
> > -    "backbone": "1.0.0 - 1.1.2",
> > -    "underscore": "1.4.4 - 1.6.0"
> > +    "underscore": "^1.8.3"
>
> I want to lock the upper version here, because underscore does not follow
> semver
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/marionettejs/backbone.syphon/pull/114/files#r34697063>
> .
>